### ISSUE TITLE
Single word predicted

### ISSUE THREAD

**Issue by goodrahstar** [Conchylicultor/DeepQA#18]
Hi @Conchylicultor ,

I was training the model for past one week and its global steps = 400k but still it is not able to predict more than one word. I used the movie corpus and few some wikipedia data to train the model.

Please let me know how will model be able to predict a sentence?

Regards,
Rahul

**Comment by Conchylicultor** [comment]
It's possible that if you train for too long, the model will overfitt and gives garbage predictions.

Try with sentence length=5 and see the results after 1 or 2 hours to see if the model seems to learn correctly. If it doesn't predict sentences longer than 1 word, something is wrong.

Otherwise, try to reclone the repository and do the above experiment to see if the problem don't come from your modifications or the added data.

**Comment by goodrahstar** [comment]
Sure will try....but I will continuously monitoring the model why running --test interactive and for none of the global steps it gave me more than word predictions.
Just for clarification on dataset, I am using dataset as:

Sent1 --> Sent2
Sent2 --> Sent3
Sent3 --> Sent4

This formation should work? If my understanding is not wrong.

**Comment by Conchylicultor** [comment]
I don't see why that shouldn't work.

You should use the `--keep_all` flag to keep track of all the intermediate
model.
Also, i suggest you to use `--test` instead of `--test interactive`, so you
don't have to manually check each model. That will generate answer to
prerecorded questions and save it in your model folder. You can also change
the questions that will be tested by modifying the text file in data/test/

**Comment by goodrahstar** [comment]
Thanks for your suggestion.. I pulled the new code and it seems to work now.It's predicting more than one words....
Just out of curiosity.. Hve you implemented attention a swell in the model?

Coz in model you have added few more param "sampled softmax (needed for attention mechanism "
but model is still using "embedding_rnn_seq2seq".

**Comment by Conchylicultor** [comment]
Indeed, attention is not implemented. I tried but unfortunately my hardware is not powerful enough. But don't hesitate to try replacing `embedding_attention_seq2seq` by `embedding_rnn_seq2seq`.

**Comment by goodrahstar** [comment]
Thanks for your valuable response. 👍
